### ISSUE TITLE
v2.3.1 released before v2.3.1_prerelease2

### ISSUE THREAD

**Issue by jschueller** [beltoforion/muparser#76]
2.3.1 was released with the final keyword that broke user parsers
then v2.3.1_prerelease2 fixed that issue
in the mean time archlinux published the broken 2.3.1
now what happens ? will 2.3.1 tag be overwritten ?
I think its best to label it 2.3.2 to clarify things, if that's ok

**Comment by beltoforion** [comment]
tl;dr;
### Yes, if 3.2.1 was published there is no other way. To finally solve this I will release 2.3.2 today. 

### The Long story:
- If 3.2.1 was published by ArchLinux they relased a version clearly marked as a prerelease. I did not expect that. I was under the assumption clearly marking it as a prerelease would prevent it from being picked up.
- 2.3.0 was released for a single day. It had the final keyword and a new bug report came in so I removed it. Not marking 2.3.0 as a prerelease was the mistake i made. In hindsight i should also not have removed it and just follow it up with 2.3.1. I did not expect it to be picked up in less than a day by any distribution. So i guess that is either an automated process or thy are really quick at updating ArchLinux.

I really hate pushing the release button but it will be resolved today. I hope...

**Comment by jschueller** [comment]
ok, thanks
